### DISCUSSION TITLE
public transport starts at 0:00

### DISCUSSION THREAD

**Discussion by Karl-1** [osmandapp/OsmAnd#18613] [Q&A]
All route planning with public transport starts at 0:00 Hours. Not "now" or adjustable, but always 0:00
The routes seem to be correct but as they are always built for 0:00 the routes only show night-lines.  After all, consistent but not usable to find a route during the day.
(I tried PT of Munich and Madgeburg)
There is also a link to the blog - but the last entry there was in 2019

Mobile phone Android, German, OSMAnd version 4.5.10
Thankj you in advance
Karl-1

**Comment by vshcherb** [comment]
That's to avoid confusion that Public Transport support times, so now it's only generic without timetable support

**Reply by Karl-1** [reply]
Just to be clear, does this mean that "public transport" is currently useless? Are there any plans to develop this further? For the moment I have switched it off.

**Reply by vshcherb** [reply]
If you need schedule yes, you can't use it. But it's very useful to have generic routes which are 90% correct during daylight time

**Reply by Karl-1** [reply]
Yes, you're probably right. I was thinking more in terms of my living environment - the big cities.  Of course, there are apps for local public transport. I don't need navigation for that. 
What really irritated me was the above-average number of night bus routes that were suggested. (My example: Munich, from Goetheplatz to Rosenheimer Platz. Normally during the day: U-Bahn->S-Bahn. about 10 minutes, no N... buses ) 
My actual question: 
It is always indicated that this part of the app is still beta. Do you know what the future plans are?
